### PR TITLE
Move iframe load handler to separate method

### PR DESCRIPTION
**Pull request by jkRoberts**
When using the DMS module linking to an existing document the edit iframe would fail to remove the loading icon. This change makes it possible for DMS javascript to manually call the iframe load logic.

### CODE REVIEW

**Comment by dhensby** [comment]
I don't see any problems with taking this in. Anyone else on the core team got an opinion?

**Comment by flamerohr** [comment]
How come some of the selectors are being changed?

If it were just splitting out loadIframe functionality, I think this is all good.

File: javascript/UploadField.js [hunk 1]
```diff
@@ -416,7 +416,7 @@
 				return false;
 			} 
 		});
-		$( 'div.ss-upload:not(.disabled):not(.readonly) .ss-uploadfield-item-edit').entwine({
+		$( 'div.ss-upload:not(.disabled):not(.readonly) .ss-uploadfield-item-edit, .ss-assetuploadfield:not(.disabled):not(.readonly) .ss-uploadfield-item-edit').entwine({
```
Comment (sminnee) [inline javascript/UploadField.js]: I don't think this should be restricted only to assetuploadfield.

**Comment by dhensby** [comment]
@jkRoberts any plans to address the feedback on this?

**Comment by maxime-rainville** [comment]
Unfortunately we're only doing security updates to SilverStripe 3 from now on, so I'll have to close this one.

If you feel this is still a worthwhile feature, please refactor it so it works with SilverStripe 4.

**Comment by maxime-rainville** [comment]
Asset management has been completely redone in SilverStripe 4. So there wouldn't be any point rebasing this PR.

**Pull request closed without merging**